packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -9,101 +9,120 @@ ouia: true
 import pfLogo from './pfLogo.svg';
 import pfLogoSmall from './pf-logo-small.svg';
 
-## Examples
 
-### Basic
+## Examples 
+
+### Basic cards
+
+Basic cards typically have a `<CardTitle>`, `<CardBody>` and `<CardFooter>`. You may omit these components as needed, but it is recommended to at least include a `<CardBody>` to provide details about the card item.
 
 ```ts file='./CardBasic.tsx'
 ```
 
-### With modifiers
+### Modifiers
+
+You can further modify the styling of the card's content by using the properties in the following example. In the example, you can enable each modifier by selecting its respective checkbox.
+
+Most modifiers can be used in combination with each other, with the exception of `isCompact` and `isLarge`, since they are contradictory.
 
 ```ts file='./CardWithModifiers.tsx'
 ```
 
-### With image and actions
+| Modifier | Description |
+| --- | --- |
+| isCompact | Modifies the card to include compact styling. Should not be used with isLarge. |
+| isFlat | Modifies the card to include flat styling. |
+| isRounded | Modifies the card to include rounded border styling. |
+| isLarge | Modifies the card to be large. Should not be used with isCompact. |
+| isFullHeight | Modifies the card so that it fills the total available height of its container. |
+| isPlain | Modifies the card to include plain styling, which removes the border and background. | 
+
+
+### Header images and actions
+
+You can include header images using `<Brand>` within `<CardHeaderMain>` and you can include header actions within `<CardActions>`.
 
 ```ts file='./CardWithImageAndActions.tsx'
 ```
 
-### Header in card head
+### Title inline with images and actions
+
+Moving `<CardTitle>` within the `CardHeader` component will style it in line with any images or actions.
 
 ```ts file='./CardHeaderInCardHead.tsx'
 ```
 
 ### Only actions in card head (no header/footer)
```

Comment:
Nit:

```suggestion
### Only actions in card header
```

I think "Only actions in card head (no header...)" could come off as contradictory. Another suggestion could be "Only actions in card header (no title/footer)"